src/ontology/edits/oeo-physical.omn

```diff
@@ -9513,6 +9519,280 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1290"@en,
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A transport network component is an artificial object that is part of a transport network."@en,
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/1267
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1297",
+        rdfs:label "transport network component"@en
+    
+    EquivalentTo: 
+        OEO_00000061
+         and (<http://purl.obolibrary.org/obo/BFO_0000050> some OEO_00320016)
+    
+    SubClassOf: 
+        OEO_00000061,
+        OEO_00020180 some OEO_00040009
+    
+    
+Class: OEO_00320022
+
+    Annotations: 
+        OEO_00040001 "http://openenergy-platform.org/ontology/oeo/oeo-physical"^^xsd:string,
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A road is a transport network component with an artificial surface that allows transport for road vehicles."@en,
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/1267
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1297",
+        rdfs:label "road"@en
+    
+    SubClassOf: 
+        OEO_00320021
+    
+    
+Class: OEO_00320023
+
+    Annotations: 
+        OEO_00040001 "http://openenergy-platform.org/ontology/oeo/oeo-physical"^^xsd:string,
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A bridge is a transport network component that spans a physical obstacle without blocking the way underneath."@en,
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/1267
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1297",
+        rdfs:comment "It can hold other transport network components such as rails and roads."@en,
+        rdfs:label "bridge"@en
+    
+    SubClassOf: 
+        OEO_00320021
+    
+    
+Class: OEO_00320024
+
+    Annotations: 
+        OEO_00040001 "http://openenergy-platform.org/ontology/oeo/oeo-physical"^^xsd:string,
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A tunnel is a transport network component that is built through a certain environment (e.g. a mountain or water) and allows to pass through that environment."@en,
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/1267
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1297",
+        rdfs:comment "It can contain other transport network components such as rails and roads."@en,
+        rdfs:label "tunnel"@en
+    
+    SubClassOf: 
+        OEO_00320021
+    
+    
+Class: OEO_00320025
+
+    Annotations: 
+        OEO_00040001 "http://openenergy-platform.org/ontology/oeo/oeo-physical"^^xsd:string,
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A railway is a transport network component that can only be used by rail vehicles."@en,
```

Comment:
```suggestion
        <http://purl.obolibrary.org/obo/IAO_0000115> "A railway is a transport network component that can only be used by trains."@en,
```

In issue #1277 we decided to finally label the class `train` and not `rail vehicle`. Therefore we should here adapt the definition of `railway`.